Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -365,43 +333,22 @@
             [NSMutableDictionary<NSString *, NSNumber *> dictionary];
         configDict[kSentryLaunchProfileConfigKeyWaitForFullDisplay] =
             @(options.enableTimeToFullDisplayTracing);
-#    if !SDK_V9
-        if ([options isContinuousProfilingEnabled]) {
-#    endif // !SDK_V9
-            if ([options isContinuousProfilingV2Enabled]) {
-                SENTRY_LOG_DEBUG(@"Configuring continuous launch profile v2.");
-                configDict[kSentryLaunchProfileConfigKeyContinuousProfilingV2] = @YES;
-                configDict[kSentryLaunchProfileConfigKeyContinuousProfilingV2Lifecycle] =
-                    @(options.profiling.lifecycle);
-                if (options.profiling.lifecycle == SentryProfileLifecycleTrace) {
-                    configDict[kSentryLaunchProfileConfigKeyTracesSampleRate]
-                        = config.tracesDecision.sampleRate;
-                    configDict[kSentryLaunchProfileConfigKeyTracesSampleRand]
-                        = config.tracesDecision.sampleRand;
-                }
-                configDict[kSentryLaunchProfileConfigKeyProfilesSampleRate]
-                    = config.profilesDecision.sampleRate;
-                configDict[kSentryLaunchProfileConfigKeyProfilesSampleRand]
-                    = config.profilesDecision.sampleRand;
-            } else {
-#    if !SDK_V9
-                SENTRY_LOG_DEBUG(@"Configuring continuous launch profile.");
-                configDict[kSentryLaunchProfileConfigKeyContinuousProfiling] = @YES;
-#    endif // !SDK_V9
+        if ([options isContinuousProfilingV2Enabled]) {
```

Comment:
`m`: I guess we should rename `isContinuousProfilingV2Enabled` to `isContinuousProfilingEnabled` as the V2 now is the default, and we don't have a V1 anymore.